sources/packages/backend/libs/sims-db/src/entities/announcement.model.ts

```diff
@@ -0,0 +1,53 @@
+import { Column, Entity, PrimaryGeneratedColumn } from "typeorm";
+import { TableNames } from "../constant";
+/**
+ * System Announcements.
+ */
+@Entity({ name: TableNames.Announcements })
+export class Announcement {
+  @PrimaryGeneratedColumn()
+  id: number;
+  /**
+   * Announcement title.
+   */
+  @Column({
+    name: "message_title",
+    nullable: false,
+  })
+  messageTitle: string;
+  /**
+   * Announcement content.
+   */
+  @Column({
+    name: "message",
+    nullable: false,
+  })
+  message: string;
+  /**
+   * Target areas for the announcement.
+   */
+  @Column("text", {
```

Comment:
Please use the column decorator the way it is used in other places. 
![image](https://github.com/user-attachments/assets/8156ec65-89b5-4aef-b54a-17e74c3f3fab)

Reply:
updated